example/build.gradle.kts

```diff
@@ -24,6 +25,9 @@ dockerCompose.isRequiredBy(project.tasks.named("run"))
 
 repositories {
     mavenCentral()
+
+    maven("https://oss.sonatype.org/content/repositories/snapshots")
```

Comment:
snapshot for now to iron out web socket WebTau API, I will release once all is good